App/Redux/CameraRollRedux.ts

```diff
@@ -43,8 +41,8 @@ export const initialState: CameraRollState = {
 }
 
 export const cameraRollSelectors = {
-  initialized: state => state.cameraRoll.initialized as boolean,
-  queriedPhotos: state => state.cameraRoll.queriedPhotos as QueriedPhotosMap
+  initialized: (state: RootState) => state.cameraRoll.initialized as boolean,
+  queriedPhotos: (state: RootState) => state.cameraRoll.queriedPhotos as QueriedPhotosMap
```

Comment:
No casts needed here now

Reply:
But now that it's typed as`RootState` they type shouldn't be `any`.

Reply:
ah, sorry, mistook what you were pointing at